<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitiativeWindow</class>
 <widget class="QWidget" name="InitiativeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InitiativeWindow</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>1051</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="stopDelayButton">
      <property name="text">
       <string>Stop all delays</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delayButton">
      <property name="text">
       <string>Delay Button</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteCreatureButton">
      <property name="text">
       <string>Delete Creature</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextCreatureButton">
      <property name="text">
       <string>Next Creature</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startEndButton">
      <property name="text">
       <string>Start combat</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1061</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#friend {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(111, 255, 0), stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox#foe {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(251, 48, 39), stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox#delaying {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(202, 203, 201), stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox#foeActive {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 191, 0), stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox#friendActive {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(23, 110, 255), stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;
}

</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1059</width>
      <height>619</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
